scripts/release

```diff
@@ -394,4 +395,48 @@ release_python_to_pypi() {
   popd
 }
 
+add_digest_to_release() {
+  local tag=$1
+  local new_image=$2
+  local image_base=$3
+  local username=$4
+  local token=$5
+  local digest=$(docker inspect --format='{{.RepoDigests}}' $new_image | sed "s|\[${image_base}@\(sha256:.*\)\]|\1|")
```

Comment:
Split this out to a dedicated function that echos the sha256 sum for a given image.